index.html

```diff
@@ -346,21 +346,23 @@ <h3>DAPT Script</h3>
           <section>
             <h4>Script Type</h4>
             <p>The <dfn>Script Type</dfn> property is a mandatory property of a <a>DAPT Script</a> which describes the type of documents used in Dubbing and Audio Description workflows, among the following: <a>Original Language Dialogue List</a>, <a>Translated Dialogue List</a>, <a>Pre-recording Dub Script</a>, <a>As-recorded Dub Script</a>, and <a>Audio Description Script</a>. <a>Original Language Dialogue List</a>, <a>Translated Dialogue List</a>, <a>Pre-recording Dub Script</a>, <a>As-recorded Dub Script</a> are referred to as <dfn>Dubbing Scripts</dfn>.</p>
-            <p>To represent this property, the following TTML structure and constraints apply:
-              <ul>
-                <li>The <code>head</code> element MUST be present, as the first child of the <code>tt</code> element</li>
-                <li>A single <code>metadata</code> element MUST be present in the <code>head</code> element with the <code>ttm:role</code> attribute set to one of the values defined below.
-                </li>
-              </ul>
+            <p>To represent this property, the following TTML attribute MUST be present on the <code>tt</code> element:</p>
+            <div class="exampleInner">
+              <pre class="nohighlight">
+  daptm:scriptType
+    : "DUBBING_ORIGINAL_DIALOGUE_LIST"
+    | "DUBBING_TRANSLATED_DIALOGUE_LIST"
```

Comment:
Looking at these values, the suffix `_LIST` looks out of place. And since we're now being explicit about it possibly containing on screen text as well as dialogue, the `DIALOGUE` bit looks odd too. Happy to open as a separate issue, but I'm leaning towards just `ORIGINAL_TRANSCRIPT` and `TRANSLATED_TRANSCRIPT`, partly because I think they are clearer and also because I suspect they are likely to be more generally useful beyond just dubbing workflows.

I'd keep `DUBBING_PRE_RECORDED` and `DUBBING_AS_RECORDED` as they are.